.script/utils/changedFilesValidator.ts

```diff
@@ -0,0 +1,48 @@
+import { GetDiffFiles } from './gitWrapper'
+import { ExitCode } from './exitCode';
+import * as logger from './logger';
+
+export type CheckOptions = {
+  onExecError(result: any, filePath: string): Promise<unknown>;
+  onCheckFile(filePath: string): Promise<ExitCode>;
+  onFinalFailed(): Promise<unknown>;
+}
+
+
+async function changedFilesValidator(checkOptions: CheckOptions, fileTypeSuffixes?: string[], filePathFolderPreffixes?: string[]){
+  
+  const changedFiles = await GetDiffFiles(fileTypeSuffixes, filePathFolderPreffixes);
+  if (changedFiles === undefined) {
+      return;
+  }
+
+  let retCode = ExitCode.SUCCESS;
+
+  for (const filePath of changedFiles) {
+    try{
```

Comment:
There is a lot of inconsistent spaces in the TS files. Please define your conventions and run a beautifier on all of them.

Reply:
Thanks! I use [prettier](https://prettier.io/docs/en/cli.html)